src/api/tooling.ts

```diff
@@ -181,6 +181,7 @@ export type CompletionsResult = {
 const {
   query,
   queryMore,
+    _ensureVersion,
```

Comment:
nit: indentation seems to be off for the added lines of code in this file

Reply:
@kyungkimsf is it possible to get this merged soon?

Reply:
> nit: indentation seems to be off for the added lines of code in this file

✅ Fixed